src/QueryEditor.tsx

```diff
@@ -334,6 +352,21 @@ export class QueryEditor extends PureComponent<Props> {
                 />
               </InlineField>
             </InlineFieldRow>
+            <InlineFieldRow>
+              <InlineField
+                  label="Instant"
+                  labelWidth={30}
+                  tooltip="Queries only first point for each series(PER PARTITION LIMIT 1)"
```

Comment:
Why do we provide this option if the user can add it directly to the query?

Reply:
It is added to Query Configurator, not to Editor.